[2024-09-16T14:22:59.398+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-09-16T14:22:59.409+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_comments_etl.extract_comments scheduled__2024-09-15T00:00:00+00:00 [queued]>
[2024-09-16T14:22:59.572+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_comments_etl.extract_comments scheduled__2024-09-15T00:00:00+00:00 [queued]>
[2024-09-16T14:22:59.573+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-09-16T14:22:59.588+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): extract_comments> on 2024-09-15 00:00:00+00:00
[2024-09-16T14:22:59.595+0000] {standard_task_runner.py:57} INFO - Started process 59 to run task
[2024-09-16T14:22:59.599+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'youtube_comments_etl', 'extract_comments', 'scheduled__2024-09-15T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/youtube_comments_dag.py', '--cfg-path', '/tmp/tmp372bln01']
[2024-09-16T14:22:59.601+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask extract_comments
[2024-09-16T14:22:59.617+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:188: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-09-16T14:22:59.660+0000] {task_command.py:410} INFO - Running <TaskInstance: youtube_comments_etl.extract_comments scheduled__2024-09-15T00:00:00+00:00 [running]> on host 9fbb4b2ec6ca
[2024-09-16T14:22:59.674+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-09-16T14:22:59.764+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_comments_etl' AIRFLOW_CTX_TASK_ID='extract_comments' AIRFLOW_CTX_EXECUTION_DATE='2024-09-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-15T00:00:00+00:00'
[2024-09-16T14:22:59.771+0000] {__init__.py:48} WARNING - file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/googleapiclient/discovery_cache/file_cache.py", line 33, in <module>
    from oauth2client.contrib.locked_file import LockedFile
ModuleNotFoundError: No module named 'oauth2client'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/googleapiclient/discovery_cache/file_cache.py", line 37, in <module>
    from oauth2client.locked_file import LockedFile
ModuleNotFoundError: No module named 'oauth2client'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/googleapiclient/discovery_cache/__init__.py", line 44, in autodetect
    from . import file_cache
  File "/home/airflow/.local/lib/python3.7/site-packages/googleapiclient/discovery_cache/file_cache.py", line 41, in <module>
    "file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth"
ImportError: file_cache is unavailable when using oauth2client >= 4.0.0 or google-auth
[2024-09-16T14:23:00.907+0000] {logging_mixin.py:149} INFO - Finished processing 20 comments.
[2024-09-16T14:23:01.167+0000] {logging_mixin.py:149} INFO - Finished processing 20 comments.
[2024-09-16T14:23:01.441+0000] {logging_mixin.py:149} INFO - Finished processing 20 comments.
[2024-09-16T14:23:01.752+0000] {logging_mixin.py:149} INFO - Finished processing 20 comments.
[2024-09-16T14:23:02.046+0000] {logging_mixin.py:149} INFO - Finished processing 13 comments.
[2024-09-16T14:23:02.046+0000] {logging_mixin.py:149} INFO - Total comments extracted: 93
[2024-09-16T14:23:02.060+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-09-16T14:23:02.069+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=youtube_comments_etl, task_id=extract_comments, execution_date=20240915T000000, start_date=20240916T142259, end_date=20240916T142302
[2024-09-16T14:23:02.104+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-09-16T14:23:02.124+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
